lib/_http_outgoing.js

```diff
@@ -667,9 +679,9 @@ function write_(msg, chunk, encoding, callback, fromEnd) {
     return true;
   }
 
-  if (!fromEnd && typeof chunk !== 'string' && !(chunk instanceof Buffer)) {
+  if (!fromEnd && typeof chunk !== 'string' && !(isUint8Array(chunk))) {
     throw new ERR_INVALID_ARG_TYPE('first argument',
-                                   ['string', 'Buffer'], chunk);
+                                   ['string', 'Buffer', 'Uint8Array'], chunk);
```

Comment:
Just to confirm `isUint8Array(new Buffer()) === true`?

Reply:
@ronag Affirmative.